frontend/src/scenes/funnels/FunnelCanvasLabel.tsx

```diff
@@ -21,20 +20,27 @@ export function FunnelCanvasLabel(): JSX.Element | null {
 
     return (
         <div className="funnel-canvas-label">
-            {allFilters.display === FUNNEL_VIZ && (
+            {allFilters.display === ChartDisplayType.FunnelViz && (
                 <>
                     <span className="text-muted-alt">Total conversion rate: </span>
-                    <span>{totalConversionRate}%</span>
+                    <span>{conversionMetrics.totalRate}%</span>
                     <span style={{ margin: '2px 8px', borderLeft: '1px solid var(--border)' }} />
                 </>
             )}
-            <span className="text-muted-alt">Average time to convert: </span>
+            {allFilters.display === ChartDisplayType.FunnelsTimeToConvert && (
+                <>
+                    <span className="text-muted-alt">Total time to convert: </span>
+                    <span>{humanFriendlyDuration(conversionMetrics.sum)}</span>
```

Comment:
Curious about the use case for this

Reply:
I think you're right it might contribute to information overload, maybe this info would be better captured in a popover?

Reply:
going to hide this